inactivity.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-parent: Hyperledger TOC
+parent: LF Decentralized Trust TAC
 nav_exclude: true
 ---
-<meta http-equiv="refresh" content="0; url=https://toc.hyperledger.org/governing-documents/inactivity.html">
-<p>The page has moved to: <a href="https://toc.hyperledger.org/governing-documents/inactivity.html">this page</a></p>
+<meta http-equiv="refresh" content="0; url=https://tac.lfdecentralizedtrust.org/governing-documents/inactivity.html">
+<p>The page has moved to: <a href="https://tac.lfdecentralizedtrust.org/governing-documents/inactivity.html">this page</a></p>
```

Comment:
I may have missed this in one of our previous TOC meetings, but are we planning to mirror `toc.hyperledger.org` to `tac.lfdecentralizedtrust.org`? Wouldn't it be better to move the non-redundant contents from those pages into this GitHUb repo instead?